exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -0,0 +1,158 @@
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class TournamentTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void justTheHeaderIfNoInput() {
+        Tournament tournament = new Tournament();
+        assertEquals("Team                           | MP |  W |  D |  L |  P", tournament.printTable());
+    }
+
+    @Test
+    public void aWinIsThreePointsALossIsZeroPoints() throws Exception {
+        Tournament tournament = new Tournament();
+        tournament.applyResults("Allegoric Alaskans;Blithering Badgers;win");
+        assertArrayEquals(new String[]{
+                "Team                           | MP |  W |  D |  L |  P",
+                "Allegoric Alaskans             |  1 |  1 |  0 |  0 |  3",
+                "Blithering Badgers             |  1 |  0 |  0 |  1 |  0"
+                }, tournament.printTable().split("\n"));
```

Comment:
Why don't you just compare the string? I think that makes it more obvious to the users what they need to implement. What do you think? :)